json2html/__init__.py

```diff
@@ -3,7 +3,7 @@
 (c) 2013 Varun Malhotra. MIT License
 '''
 
-from jsonconv import *
+from .jsonconv import *
 
 __author__ = 'Varun Malhotra'
 __version__ = '1.0.3'
```

Comment:
This PR has a lot of good things in it. I would consider it a minor release.
How about changing the version to `1.1.0` in `__init__.py`, `setup.py` and `test/__init__.py`? 😄

Your hard work should be valued.